OWNERS_ALIASES

```diff
@@ -7,11 +7,23 @@ aliases:
   - BeckerMax
   - ialidzhikov
   - kris94
+  - oliver-goetz
   - plkokanov
   - rfranzke
   - shafeeqes
   - timebertt
   - timuthy
+  - abdasgupta
+  - danielfoehrKn
+  - DockToFuture
+  - istvanballok
+  - Kristian-ZH
+  - ScheererJ
+  - shreyas-s-rao
+  - vlvasilev
+  - voelzmo
+  - vpnachev
+  - wyb1
   gardener-approvers:
   - acumino
   - ary1992
```

Comment:
Drop  `BeckerMax` from approvers

Reply:
And maybe add `petersutter` ?

Reply:
Good call, done :) 
@petersutter welcome to the party 😄